### PR TITLE
Position tracker refactor post serialization

### PR DESCRIPTION
**Pull request by dalejung**
WIP until I get the corresponding qexec branch updated. 

Oh, i need to generate new states files?

### CODE REVIEW

File: tests/test_versioning.py [hunk 1]
```diff
@@ -80,30 +50,12 @@ def load_state_from_disk(self, cls):
...
-        (Transaction,
-            (8554, 10, datetime.datetime(2013, 6, 19), 100, "0000"), 'dict'),
-        (VolumeShareSlippage, (), 'dict'),
-        (Account, (), 'dict'),
-        (Portfolio, (), 'dict'),
-        (ProtocolPosition, (8554,), 'dict')
-    ])
+    @parameterized.expand(object_serialization_cases(skip_daily=True))
     def test_object_serialization(self,
+                                  _,
```
Comment (ehebert) [inline tests/test_versioning.py]: Can we change how this is being called, instead of adding an ignored input?

Comment (dalejung) [inline tests/test_versioning.py]: mm afaik no, not unless we want to go back to nonuseful test case names.

Comment (dalejung) [inline tests/test_versioning.py]: err, looks like newer releases allow for this?

Comment (ehebert) [inline tests/test_versioning.py]: Ah, I did not realize it was for support of the parameterized naming.

Comment (dalejung) [inline tests/test_versioning.py]: lol yeah. all of those test_balh_blah_0 was driving me crazy

**Comment by ehebert** [comment]
Looks good to me.

Going to merge pending a few internal test suite runs.

**Comment by ssanderson** [comment]
@ehebert I'm assuming this is no longer viable given all the changes on your and @jbredeche's branch?

**Comment by ehebert** [comment]
Correct, closing.

**Pull request closed without merging**